spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-data-redis/src/main/java/smoketest/data/redis/SampleRedisApplication.java

```diff
@@ -40,8 +41,10 @@ public void run(String... args) throws Exception {
 	}
 
 	public static void main(String[] args) {
+		ConfigurableApplicationContext context = new SpringApplicationBuilder(SampleRedisApplication.class)
+				.properties("spring.redis.lettuce.pool.enabled:false").run(args);
 		// Close the context so it doesn't stay awake listening for redis
-		SpringApplication.run(SampleRedisApplication.class, args).close();
+		context.close();
```

Comment:
If the context does not properly shutdown when there is a pool, we may need to double check that we can't do anything about it.